blackjax/ns/nss.py

```diff
@@ -175,10 +123,7 @@ def compute_covariance_from_particles(
         This means each leaf of `cov_pytree` will have a shape `(D, *leaf_original_dims)`.
     """
     cov_matrix = jnp.atleast_2d(particles_covariance_matrix(state.particles))
-    single_particle = get_first_row(state.particles)
-    _, unravel_fn = ravel_pytree(single_particle)
-    cov_pytree = jax.vmap(unravel_fn)(cov_matrix)
-    return {"cov": cov_pytree}
+    return {"cov": cov_matrix}
```

Comment:
could possibly make it a one-liner

Reply:
and also possibly reduce docstring length for consistency with the rest of blackjax?